modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,8 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {ProductShortName} fixed issues
 
+* link:https://issues.redhat.com/browse/OSSM-419[OSSM-419] Namespaces with similar names will all show in Kiali namespace list, even though namespaces may not be defined in Service Mesh Member Role.
```

Comment:
Hi, @neal-timpe. This looks like a description of the bug from OSSM-419. I'm not familiar with how ossm handles release notes. However, a typical release notes describes the bug behavior in past tense, followed by a description of the fix in the present tense. For example, see https://docs.google.com/presentation/d/1TaI1jbV-dhYueUfElPjS_HEqLag_HexFWp85Kdst4y8/edit#slide=id.gc78664d2cb_1_47

Reply:
A naive reading of "Namespaces with similar names will all show in Kiali namespace list" under the `== {ProductShortName} fixed issues` heading suggests that you're describing the new fixed behavior.